packages/test-apps/react-vite/src/components/ExpEditorMode.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface ExpEditorModeProps {
+  isInExpEditorMode?: boolean;
+  myValue: string;
+  className?: string;
+}
+
+const ExpEditorMode: React.FC<ExpEditorModeProps> = (props) => {
+  const { isInExpEditorMode, myValue, className } = props;
+  console.log('eee', { props });
+  return (
+    <section className={className}>
+      <div>isInExpEditorMode {isInExpEditorMode ? 'true' : 'false'}</div>
+      <div>myValue {myValue}</div>
+    </section>
+  );
+};
```

Comment:
Do you intentionally left this testing component here or do you want that to get merged as well?

Reply:
no probably not, nice catch